docs/source/sensors/Chandrayaan.rst

```diff
@@ -0,0 +1,5 @@
+Chandrayaan1 M3
+===============
```

Comment:
Consider spelling out these short forms? So, this would be Chandrayaan1 Moon Mineralogy Mapper (M3) or something similar? Why? Because that is the missions official non-acronym name and users may be familiar with it as such.